Client/Frontend/Browser/TopTabsViewController.swift

```diff
@@ -183,13 +188,16 @@ class TopTabsViewController: UIViewController {
     }
     
     func scrollToCurrentTab(animated: Bool = true) {
+        if !animated {
+            collectionView.contentOffset = CGPoint.zero
```

Comment:
I don't understand why it's necessary to set the `contentOffset` to zero only in the non animated case. The `animated` boolean should only toggle if the animation occurs or not and I think that's handled by the `scrollRectToVisible` method calls below.